docker-compose.test.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+services:
+  continuum-auth:
+    image: libero/continuum-auth:${IMAGE_TAG:-local}
+    container_name: test_continuum_auth
+    ports:
+      - "${AUTHENTICATION_PORT:-3001}:3001"
+    environment: 
+      CONFIG_PATH: /etc/reviewer/config.json
+    volumes:
+      - ./tests/config/continuum-auth.json:/etc/reviewer/config.json
+    healthcheck:
```

Comment:
should be in the Dockerfile itself?

Reply:
I've fixed the Dockerfile `HEALTHCHECK` configuration but because of logging we have it set to a `1m` intervals for production image. To keep CI fast I'm reducing that to `10s` for the integration tests to get started quicker (we're using the `.scripts` wait-healthy script). Couldn't find a way to only reconfigure the interval argument through a docker compose, looks like defining healthcheck means you need to re-define the command, timeout and retries too.